docs/testnet/tutorial.md

```diff
@@ -0,0 +1,158 @@
+
+# Let's go
+This 'Guessing Game' tutorial provides an introduction to the fundamentals of Obscuro and shows you how to:
+- Set up your local Ethereum development environment for Obscuro.
+- Use Obscuro concepts to develop a dApp
+
+What is the *Guessing Game*?
+
+The Guessing Game is a dApp that you will build that demonstrates a basic Obscuro use case, a simple number guessing game. The contract generates a random secret number, which is never revealed to anyone, including node operators, developers or end-user because of the privacy provided by Obscuro. The goal of the game is to guess this number, and each time an attempt is made, an entrance fee of 1 token is paid. If a user correctly guesses the number, the contract will pay out all of the accumulated entrance fees to them and reset itself with a new random number.
+
+Without Obscuro, it would be possible to look up the internal state of the contract and cheat, and the game wouldn't work.
+
+The dApp has many of the features that you'd expect to find, including:
+- Generating a random number known only to the dApp
+- Allow users to guess the numbers by depositing a token for each play
+- Colder/Warmer functionality to help guide users to the correct number
+- Events to alert users whether they have won or not
+- Track all user's plays
+- Contract owner privileges to start and stop the game
+
+# Set up your environment
+- You'll need to install MetaMask following the instructions [here](https://metamask.io/)
+- Then download and run the Obscuro Wallet extension following the instructions [here](https://docs.obscu.ro/wallet-extension/wallet-extension.html)
+- Now configure MetaMask following the instructions [here](https://docs.obscu.ro/wallet-extension/configure-metamask.html)
+- Finally, check you can open the Remix IDE by visiting the following URL in your browser https://remix.ethereum.org
+
+That's it. You're all set to start building your first dApp on Obscuro.
+
+## 1. To begin, we'll clone a simple repo from Github through Remix by following these instructions
+
+#### Click on GitHub
+![Step 1 screenshot](https://images.tango.us/workflows/57863807-6b96-4295-a7a1-42da1623f030/steps/919b227f-f1f5-4af3-aba1-7fb83576d505/8144f3f5-7855-4c83-86dd-c88e5476c367.png?crop=focalpoint&fit=crop&fp-x=0.5000&fp-y=0.5000&fp-z=1.0000&w=1200&mark-w=0.2&mark-pad=0&mark64=aHR0cHM6Ly9pbWFnZXMudGFuZ28udXMvc3RhdGljL21hZGUtd2l0aC10YW5nby13YXRlcm1hcmsucG5n&ar=874%3A594)
+
+
+#### Paste "https://github.com/obscuronet/sample-applications/blob/main/number-guessing-game/contracts/Guess.sol" into input
+![Step 2 screenshot](https://images.tango.us/workflows/57863807-6b96-4295-a7a1-42da1623f030/steps/4b3b212d-2518-4cc9-aab2-4befe128b38c/66a9a87d-0899-42d2-8f6a-98f204b25825.png?crop=focalpoint&fit=crop&fp-x=0.5000&fp-y=0.5000&fp-z=1.0000&w=1200&mark-w=0.2&mark-pad=0&mark64=aHR0cHM6Ly9pbWFnZXMudGFuZ28udXMvc3RhdGljL21hZGUtd2l0aC10YW5nby13YXRlcm1hcmsucG5n&ar=530%3A371)
+
+
+#### Select Guess.sol, and we're ready
```

Comment:
I just tried it, and the file window didn't autoexpand to show the contract (it did the last time, weirdly). Worth adding a step to navigate to the file, just in case?